common/app/controllers/EmailSignupController.scala

```diff
@@ -51,6 +51,27 @@ case class EmailForm(
     }
 }
 
+case class EmailFormManyNewsletters(
+    email: String,
+    listNames: Seq[String],
+    referrer: Option[String],
+    ref: Option[String],
+    refViewId: Option[String],
+    campaignCode: Option[String],
+    googleRecaptchaResponse: Option[String],
+    name: String,
+) {
+
+  // `name` is a hidden (via css) form input
+  // if it was set to something this form was likely filled by a bot
+  // https://stackoverflow.com/a/34623588/2823715
+  def isLikelyBotSubmission: Boolean =
```

Comment:
If we already use recaptcha why do we need our own bot detection code?

Reply:
Pushed the changes to take this out.